### DISCUSSION TITLE
How to convert existing openapi.yml file to TypeSpec

### DISCUSSION THREAD

**Discussion by luistrigueiros** [microsoft/typespec#3449] [Q&A]
Is there a way to convert one existing openapi.yml file to TypeSpec as why of bootstrap and start ?
Suppose one has several existing files that would like to convert to bootstrap the project ?

Thank you, kindly
  Luis

**Comment by timotheeguerin** [comment]
We do not have this yet but is in our plans, see this issue https://github.com/microsoft/typespec/issues/3038 for tracking.

**Comment by kolypto** [comment]
Already implemented: 

```
$ npm install -g @typespec/openapi3
$ tsp-openapi3 openapi.legacy.yaml --output-dir ./tsp-convert
```

See: https://typespec.io/docs/emitters/openapi3/cli/
